frontend/src/libraries/explorer-wamp/accounts.ts

```diff
@@ -82,6 +89,24 @@ export default class AccountsApi extends ExplorerApi {
     paginationIndexer?: AccountPagination
   ): Promise<PaginatedAccountBasicInfo[]> {
     try {
+      if (this.selectOption === "Legacy") {
+        return await this.call("select", [
+          `SELECT account_id as accountId, created_at_block_timestamp as createdAtBlockTimestamp, 
+            created_by_transaction_hash as createdByTransactionHash, account_index as accountIndex
+            FROM accounts
+            ${
+              paginationIndexer
+                ? `WHERE created_at_block_timestamp < :endTimestamp OR (created_at_block_timestamp = :endTimestamp AND account_index < :accountIndex)`
+                : ""
+            }
+            ORDER BY created_at_block_timestamp DESC, account_index DESC
+            LIMIT :limit`,
+          {
+            limit,
+            ...paginationIndexer,
+          },
+        ]);
+      }
       return await this.call("select", [
         `SELECT account_id as accountId, created_at_block_timestamp as createdAtBlockTimestamp, 
           created_by_transaction_hash as createdByTransactionHash, account_index as accountIndex
```

Comment:
The schema on Indexer is different from the one you use here. Please, double-check that this is even executed.

Reply:
I know, and it is not easy as it changed a lot other than blocks, I will think a good solution here